src/index.html

```diff
@@ -12,10 +12,46 @@
         minimum-scale=1.0
       "
     >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" 
+    rel="stylesheet">
+  
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__item-macspecs"
+      src="/images/imac.jpeg" 
+      alt="imac">
+      <div class="card__item-macspecs-text">
+      APPLE A1419 iMac 27" Retina 5k Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__item-prdcode">
+        Product code: 195434
+      </div>
+      <div class="stars stars--4">
```

Comment:
Add some space between blocks like these
```suggestion

      <div class="card__item-macspecs-text">
        APPLE A1419 iMac 27" Retina 5k Monoblock (MNED2UA/A)
      </div>
      
      <div class="card__item-prdcode">
        Product code: 195434
      </div>
      
      <div class="stars stars--4">
```